src/main/java/sprouts/Val.java

```diff
@@ -683,9 +682,9 @@ default boolean is( @Nullable T otherItem ) {
 	 * @param other The other property of the same type as is wrapped by this.
 	 * @return The truth value determining if the item of the supplied property is equal to the wrapped item.
 	 */
-	default boolean is( Val<T> other ) {
+	default boolean is( Val<@Nullable T> other ) {
```

Comment:
As far as I understand, yes. This means that even if `T` is not nullable, we can call it with a nullable version of it. Otherwise, we would not be able to call it with a nullable version if `T` is not nullable. The other way around should always be possible. (`String` can be used wherever `@Nullable String` is expected)

It should behave like this:
```Java
Var<String> s1 = Var.of("x");
Var<String> s2 = Var.of("x");
Var<@Nullable String> s3 = Var.ofNullable(String.class, "x");

s1.is(s3); // true
s3.is(s1); // true
s1.is(s2); // true
```

Reply:
Ah I see! Yeah that makes sense.
I just tested this locally and it all compiled.